<launch>
  <arg name="bag_path"/>
  <arg name="vehicle_model" default="lexus"/>
  
  <include file="$(find-pkg-share global_parameter_loader)/launch/global_params.launch.py">
    <arg name="vehicle_model" value="$(var vehicle_model)"/>
  </include>

  <group>
	<push-ros-namespace namespace="planning"/>
	<group>
	  <push-ros-namespace namespace="scenario_planning"/>
	  <node pkg="trajectory_optimizer" exec="benchmark_node" name="benchmark">
		<param from="$(find-pkg-share autoware_launch)/config/planning/scenario_planning/common/common.param.yaml"/>
		<param from="$(find-pkg-share autoware_launch)/config/planning/scenario_planning/common/nearest_search.param.yaml"/>
		<param from="$(find-pkg-share autoware_launch)/config/planning/scenario_planning/common/motion_velocity_smoother/motion_velocity_smoother.param.yaml"/>
		<param from="$(find-pkg-share autoware_launch)/config/planning/scenario_planning/common/motion_velocity_smoother/JerkFiltered.param.yaml"/>
		<param name="bag_path" value="$(var bag_path)"/>
	  </node>
	</group>
  </group>
</launch>
